### PR TITLE
DEP move to the travis env var api for tokens

### LINKED ISSUE #1245: deprecate the binstar token in the conda-forge.yml file
Travis now has tokens stored directly. We should remove any use of the encrypted token in the file.

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Added a ``news`` entry

Closes #1245

### CODE REVIEW

**Comment by scopatz** [comment]
Thanks! is there a test that you can add for this?

**Comment by beckermr** [comment]
Not against the love Travis api. We have very few tests for ci registration because of this. Any ideas without using mocks that make the test a bit of a tautology.

**Comment by isuruf** [comment]
Can we only deprecate this instead of removing? I have some external feedstocks that use the travis token.

File: news/travis.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+* Added feature to upload the BINSTAR_TOKEN for travis-ci.com directly
+  through the API
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* Removed the BINSTAR_TOKEN from the conda-forge.yml for travis-ci.com
```
Comment (isuruf) [inline news/travis.rst]: This is not a deprecation in the current code. It's a removal.

**Comment by beckermr** [comment]
Good catch @isuruf! This would have broken existing feedstocks otherwise. I think it is fixed now!

**Pull request merged**